tasks/SmoothieChef.tex

```diff
@@ -0,0 +1,29 @@
+\section{Smoothie Chef [Party Host]}
+The home owner will teach the robot to make a smoothie which guests at the party really like!
+
+\subsection{Focus}
+Learning from demonstration.
+
+\subsection{Main Goal}
+The home owner will teach the robot using demonstrations how to make a smoothie. The robot is then to repeat the task. A designated team member will show the home owner how to perform the demonstration. To avoid liquids and blenders, we will have the robot place the fruit corresponding to the desired smoothie into a bowl.
```

Comment:
Main goal should be short. Maybe smth like: "The robot selects ingredients for a smoothie for which it learned the recipe."